src/backend/joanie/core/api/admin.py

```diff
@@ -308,3 +309,32 @@ class ContractDefinitionViewSet(viewsets.ModelViewSet):
     permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
     serializer_class = serializers.AdminContractDefinitionSerializer
     queryset = models.ContractDefinition.objects.all()
+
+
+class NestedProductOrderGroupViewSet(
+    NestedGenericViewSet,
+    mixins.CreateModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.ListModelMixin,
+):
+    """
+    OrderGroup ViewSet
+    """
+
+    authentication_classes = [SessionAuthenticationWithAuthenticateHeader]
+    permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
+    serializer_class = serializers.AdminOrderGroupSerializer
```

Comment:
When time comes and we need more than one serializer, i have seen this practice often to define a dictionary that maps actions to their corresponding serializers :
```suggestion
serializer_classes = {
        "create": serializers.AdminOrderGroupCreateSerializer,
        "retrieve": serializers.AdminOrderGroupSerializer,  
        # Add more actions as needed (list, update, partial_update...)
    }
```